.github/workflows/ci_cd.yml

```diff
@@ -101,6 +100,15 @@ jobs:
           python-version: ${{ env.MAIN_PYTHON_VERSION }}
           use-python-cache: false
 
+  check-licenses:
```

Comment:
We can not add this action within the CI pipeline of this repository. The reason is that this is not a Python package. Even if we use the `skip-install`, we are not creating any Python virtual environment or publishing third party packages.

This is the reason of the failing CI.